user_info.go

```diff
@@ -30,3 +30,11 @@ func (c *WorkwxApp) ListUsersByDeptID(deptID int64, fetchChild bool) ([]*UserInf
 	}
 	return users, nil
 }
+
+// UseridByMobile 通过手机号获取 userid
+func (c *WorkwxApp) UseridByMobile(mobile string) (string, error) {
+	resp, err := c.execUseridByMobile(reqUseridByMobile{
+		Mobile: mobile,
+	})
+	return resp.Userid, err
```

Comment:
According to this library's coding convention, response and err are never returned together. Please use the `if err != nil` logic like surrounding code, even if it seems pointless.

Reply:
fixed